pkg/apis/core/v1beta1/types_shoot.go

```diff
@@ -457,6 +457,11 @@ type KubeAPIServerConfig struct {
 	// of the kube-apiserver.
 	// +optional
 	ServiceAccountConfig *ServiceAccountConfig `json:"serviceAccountConfig,omitempty" protobuf:"bytes,8,opt,name=serviceAccountConfig"`
+	// WatchCacheSizes contains configuration of the API server's watch cache sizes.
+	// Starting from kubernetes v1.19, the API server's watch cache size is adapted dynamically and setting the watch
+	// cache size flags will have no effect, except when setting it to 0 (which disables the watch cache).
+	// +optional
+	WatchCacheSizes *WatchCacheSizes `json:"watchCacheSizes,omitempty" protobuf:"bytes,9,opt,name=watchCacheSizes"`
```

Comment:
do you think it makes sense to add user documentation for this?

Reply:
Hmm, TBH, I think the comments here (which will also be transported to the API reference) are enough documentation on our side. I see, the topic is rarely documented on k/k side, but I would rather see it there...